charts/MapTab.tsx

```diff
@@ -75,7 +75,7 @@ class MapWithLegend extends React.Component<MapWithLegendProps> {
     }
 
     // Determine if we can go to line chart by clicking on a given map entity
-    isEntityClickable(featureId: string | number | undefined) {
+    private isEntityClickable(featureId: string | number | undefined) {
         if (
             !this.context.chart.hasChartTab ||
             !this.context.chart.isLineChart ||
```

Comment:
Turns out this doesn't work for scatter plots (`Align outbreaks`), which threw me off in the Data Explorer.
We probably want to add that one too?

Reply:
Yes, @shaahmad I think was going to make that change to also enable the sparklines on aligned charts.

Reply:
Okay I made this change as well so scatterplots now have the same map tooltip behavior:

Seems to be an improvement on the data explorers as well as most of the Scatter+Map charts (http://localhost:3030/admin/test/embeds?type=ScatterPlot&tab=map).

Some of the Scatter+Maps showing the sparkline might not make sense if there is only 1 value, but that could be improved in a later PR if needed.

Reply:
In regards to whether we should add Shift+click help text, I'm not sure. We could see the analytics on map clicks in general, then see shift clicks, then could change the text if we wanted to boost that. I think it might help but it also might slightly confuse users that just want to click, as it requires them to change tabs to see the relevant change.